### PR TITLE
Add looker studio to ownership

### PR DESCRIPTION
**Pull request by mostafa-anm**
Handed over from Bianka to me.

### CODE REVIEW

File: ownership.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Besides transparency, this might come in handy in case of vacations or sickness.
 | Storyboard via [Miro][Miro]          |            | x                 |              |          |              |
 | Designs via [Figma][Figma]           |            | x                 |              |          |              |
 | Google Tag Manager                   |            |                   | x            |          |              |
+| Looker Studio                        |            |                   | x            |          |              |
```
Comment (herrbenesch) [inline ownership.md:18]: A link would be nice :)

Comment (mostafa-anm) [reply ownership.md]: Good point, done! :)

**Review by herrbenesch — APPROVED** [review]
👍

**Comment by mostafa-anm** [comment]
@herrbenesch Thanks!!

**Pull request merged**